ReactSkia/sdk/OnScreenKeyBoard.cpp

```diff
@@ -468,12 +493,41 @@ void OnScreenKeyboard ::drawHighLightOnKey(SkPoint index) {
 }
 
 void OnScreenKeyboard::onHWkeyHandler(rnsKey keyValue, rnsKeyAction eventKeyAction) {
-  RNS_LOG_DEBUG(__func__<<"rnsKey: "<<RNSKeyMap[keyValue]<<" rnsKeyAction: "<<((eventKeyAction ==0) ? "RNS_KEY_Press ": "RNS_KEY_Release ")<<eventKeyAction );
+  RNS_LOG_DEBUG("rnsKey: "<<RNSKeyMap[keyValue]<<" rnsKeyAction: "<<((eventKeyAction ==0) ? "RNS_KEY_Press ": "RNS_KEY_Release ") );
+
   if(eventKeyAction == RNS_KEY_Release) {
-    NotificationCenter::subWindowCenter().emit("onOSKKeyEvent", keyValue, eventKeyAction);
+#if ENABLE(FEATURE_KEY_THROTTLING)
+    if(onKeyRepeatMode_) {
+      if(!repeatKeyQueue_->isEmpty()) {
+          repeatKeyQueue_->clear();
+      }
+    }
+    previousKey_ = RNS_KEY_UnKnown;
+    onKeyRepeatMode_ = false;
+#endif /*ENABLE_FEATURE_KEY_THROTTLING*/
+    NotificationCenter::subWindowCenter().emit("onOSKKeyEvent", keyValue, RNS_KEY_Release);
```

Comment:
should this release key action be sent for previousKey ?
because we will be sending Press event for processed key and release with unprocessed key.

Reply:
This Release is a common Release for OSKkeys. PSK keys are generated in press and any release receives from window will be sent back